### PR TITLE
Implement automatic alerts handling in scope of a session

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The solution is still not ideal, but it works for the most popular usage patterns.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Can you please take a look?

File: WebDriverAgentLib/Utilities/FBAlertsMonitor.m [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+@property (atomic) BOOL isMonitoring;
+
+@end
+
+@implementation FBAlertsMonitor
+
+- (instancetype)init
+{
+  if ((self = [super init])) {
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Utilities/FBAlertsMonitor.m:28]: `(self = [super init])`?

**Review by KazuCocoa — APPROVED** [review]
🙏

File: WebDriverAgentLib/Utilities/FBAlertsMonitor.m (hunk 1 above)
Comment (mykola-mokhnach) [reply WebDriverAgentLib/Utilities/FBAlertsMonitor.m:28]: yep, what's wrong with it?

Comment (mykola-mokhnach) [reply WebDriverAgentLib/Utilities/FBAlertsMonitor.m:28]: I do put brackets to emphasise assigning a variable inside if

**Pull request merged**